supereeg/helpers.py

```diff
@@ -219,12 +223,13 @@ def aggregate(p, n):
         return p + n
 
     def zcorr_xform(bo):
-        return np.multiply(bo.n_secs, _r2z(1 - squareform(pdist(bo.get_data().T, 'correlation'))))
-
+        return np.multiply(bo.dur, _r2z(1 - squareform(pdist(bo.get_data().T, 'correlation'))))
+        ### ask jeremy about this!
+        ## what if bo.nsecs is 0 or None (issue arising in NIFTI case)
```

Comment:
I think this is resolved now that we assume `samplerate==1` for `Nifti` objects-- if so, remove the above comments.  (Or if not resolved, what are the remaining issues?)